README.md

```diff
@@ -1,4 +1,16 @@
 # cypress-docs-pt-br
 
-* [Por que Cypress?](./why-cypress.md)
-* [Diferenças-chave](./key-differences.md)
+## Documentação oficial
+<br/>
```

Comment:
Why this `<br/>` tag? I'd prefer without it.

Reply:
I just added this tag to add more space between sections. It's not a good practice using `<br/>`? If yes, I'll remove it

Reply:
I simply don't think it's necessary.

Reply:
@wlsf82 I removed from my last commit